README.md

```diff
@@ -67,6 +67,7 @@ As of 2.0.0, errors that fail to send to Raygun will be logged to the standard W
 
 Changelog
 ---------
+- 2.0.1: Bump Raygun4PHP dependency to v2.3.1 to improve support for PHP version 8.2
```

Comment:
Do we need to say improve support vs., just support. It reads watered down and weak